crates/ruff/src/args.rs

```diff
@@ -131,6 +132,7 @@ pub struct CheckCommand {
     ignore_noqa: bool,
 
     /// Output serialization format for violations.
+    /// The default serialization format is "concise".
```

Comment:
Nit: should we document the preview difference? (Up to you, not a strong opinion.)